configuration/nfs.md

```diff
@@ -0,0 +1,303 @@
+# NFS - Network File System
```

Comment:
The page heading should be the same as the link text from the main page. So "Network File System".